src/components/loginForm/index.js

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Button from "../../commons/buttons";
+
+const Login = (props) => {
+  const {
+    onSubmit, errors, onRegister, onPassword,isProcessing 
+  } = props;
+  const emailError = errors ? errors.email : "";
+  const passwordError = errors ? errors.password : "";
+  const header = "Author's Haven";
+  return (
+    <div className="login-wrapper">
+      <div className="col-md-12">
+        <h3>{header}</h3>
+        <form onSubmit={onSubmit}>
+          <div className="form-group">
+            <label htmlFor="email">Email Address</label>
+            <input type="email" name="email" className="form-control" required placeholder="enter email" />
+            <div><p className="text-danger">{emailError}</p>
```

Comment:
Thanks for the good work @wasibani-roy ,there are some linting issues here,The p should be on  a new line
We could re write this part as
 ```  
     <div>
             <p className="text-danger">{emailError}</p>
     </div>
```